### ISSUE TITLE
Volume Amount Actions

### ISSUE THREAD

**Issue by sjyMystery** [palantir/gradle-docker#322]
## What happened?

Absolutely, Docker Volume Path SHOULD NEVER BE THE ABSOLUTE PATH.

It's because somebody may need to load data from labeled volumes on the docker-machine or remote docker dameon

For example, if i want to mount a remote volume from the remote docker service , the absolute path will prevent me from my target.

## What did you want to happen?

Use Relative Path In -v instead , or provide more options for others.

**Comment by djechelon** [comment]
Related: #138

I was going to write a new issue but I can join this.

MY problem is that I can't run Postgres on Windows host (related: https://github.com/docker/for-win/issues/39). This is a widely known problem with Windows hosts

**Comment by alexey-sh** [comment]
@djechelon did you fix your issue?

**Comment by djechelon** [comment]
No. In fact I never ran that container via Gradle docker plugin. I had to start from command line
